winsock-demo/winsock-client/Client/Client.cpp

```diff
@@ -0,0 +1,116 @@
+#include "Client.h"
+
+void Client::startClient() {
+    loadWinsockLibrary();
+    createHintsForAddress();
+    retrieveAddressInfo();
+    connectToTheServer();
+}
+
+void Client::handleRequestsAndResponses() {
+    while(sendMessage() == 0) {
+        receiveMessage();
+    }
+}
+
+void Client::closeClient() {
+    terminateSession(&connectSocket, NULL);
+}
+
+void Client::loadWinsockLibrary() {
+    if (WSAStartup(MAKEWORD(2, 2), &wsaData) != 0)
+    {
+        cout << "An error occured while initiating use of Winsock library on the client" << endl;
+        exit(1);
+    }
+}
+
+void Client::createHintsForAddress() {
+    ZeroMemory(&hints, sizeof(hints));
+    hints.ai_family = AF_INET;
+    hints.ai_socktype = SOCK_STREAM;
+    hints.ai_protocol = IPPROTO_TCP;
+}
+
+void Client::retrieveAddressInfo() {
+    if (getaddrinfo("localhost", PORT_NUMBER, &hints, &result) != 0)
+    {
```

Comment:
code style notice: 
lets use the same brackets stylels for all if\else, for, and etc